cli/commands/preview/create.ts

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import archiver from 'archiver';
+import fetch from 'node-fetch';
+import { BaseCommand, OutputFormat } from 'cli/commands/base';
+
+// This is DUMMY code for now. It's only meant as a reference for the actual implementation.
+export class PreviewCreateCommand extends BaseCommand {
+	private folder: string;
+	private archivePath: string;
+
+	protected readonly STATUSES = {
```

Comment:
Why do all command needs "statuses"?

Reply:
My ambition here was to add a baseline for printing well-structured standard output to be consumed by Studio. Having a statically defined list of known statuses seems like a good part of that strategy.